installers/go-server/release/go-server.init

```diff
@@ -54,13 +54,13 @@ check_proc() {
 
 start_go_server() {
     if [ "${CUR_USER}" != "root" ] && [ "${CUR_USER}" != "go" ]; then
-        log_failure_msg "Go Server can only be started as 'root' or 'go' user."
+        log_failure_msg "GoCD Server can only be started as 'root' or 'go' user."
         exit 4
     fi
 
     check_proc
     if [ $? -eq 0 ]; then
-        log_success_msg "Go Server already running on http://$HOSTNAME:$GO_SERVER_PORT/go"
+        log_success_msg "GoCD Server already running on http://$HOSTNAME:$GO_SERVER_PORT/go and https://$HOSTNAME:$GO_SERVER_SSL_PORT/go"
```

Comment:
Right. I saw that. However, if GO_SERVER_PORT can be assumed to exist, then GO_SERVER_SSL_PORT should be too. If not, both should change. Right? The only option I can see apart from this is to remove the whole message.